extensions/amp-video/0.1/amp-video.js

```diff
@@ -804,7 +813,21 @@ class AmpVideo extends AMP.BaseElement {
     if (!this.hideBlurryPlaceholder_()) {
       this.togglePlaceholder(false);
     }
-    this.removePosterForAndroidBug_();
+    // if (!Services.platformFor(this.win).isAndroid()) {
+    //   return;
+    // }
```

Comment:
@gmajoulet We can check before making all these listeners that the platform is Android (kind of a premature optimization), because the actual code we want to run will only work for Android anyways. Does it make sense to keep or remove?